### PR TITLE
Essay Proposal - Azure

### PR DESCRIPTION
**Pull request by johanmallobakken**
# Essay Proposal
I want to write an essay on Azure DevOps Services. 

The essay will cover:
- Intro to DevOps
- What is Azure DevOps?
- In short the background/history 
- Why use Azure DevOps?
- How to use Azure DevOps?
- Services and tools and how they can be applied to the different phases in the application lifecycle.  (Boards, Pipelines, Repos, Artifacts, Testing)
- How Azure DevOps and it's services compare to other tools and services (e.g. Jenkins and Azure DevOps Pipelines, JIRA and Azure DevOps Boards, GitLab and Azure DevOps)
- Conclusion/Summary

## Members
Johan Mallo Bakken <johanmb@kth.se> [johanmb](https://github.com/johanmallobakken)

### CODE REVIEW

File: essay_proposal.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+# Essay Proposal
+I want to write an essay on Azure DevOps Services. 
+
+The essay will cover:
+- What is it?
+- Why use it?
+- How to use it?
+- Services and tools (Boards, Pipelines, Repos, Artifacts, Testing)
+- How it compares to other tools like Jenkins, JIRA, GitLab
```
Comment (cesarsotovalero) [inline essay_proposal.md]: Azure is an IaS provider. It is not comparable to Jenkins, JIRA, GitLab.

**Review by cesarsotovalero — CHANGES_REQUESTED** [review]
Please add more details of your proposal.

File: essay_proposal.md (hunk 1 above)
Comment (johanmallobakken) [reply essay_proposal.md]: What I meant is to compare Azure DevOps Pipelines with Jenkins and Azure DevOps Boards with JIRA and Azure DevOps with Gitlab. Is this not possible?

**Review by cesarsotovalero — APPROVED** [review]

**Comment by cesarsotovalero** [comment]
@johanmallobakken  LGTM, go ahead with the essary.

**Comment by hassbring** [comment]
Would like to give feedback. PR #1154.

**Pull request merged**